B827EBFFFE78569C.json

```diff
@@ -0,0 +1,19 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "b827ebfffe78569c",
```

Comment:
Could you make this upper case, please? Thx!

Reply:
Dear gonzalocasas,
Thank a lot. It's done. Enjoy!